js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_ja.properties

```diff
@@ -1142,3 +1142,9 @@ authnContextClassRefsHelp=要求されたAuthnContextのClassRefの順序付き
 useLowerCaseBearerType=トークンレスポンスで小文字のベアラータイプを使用する
 confirmPasswordDoesNotMatch=パスワードとパスワード確認が一致しません。
 unmanagedAttributesHelpText=管理されていない属性は、ユーザープロファイルの設定で明示的に定義されていないユーザー属性です。 デフォルトでは、管理されていない属性は `Disabled` であり、登録、アカウント、管理コンソールなどの任意のコンテキストから利用できません。 `有効` を設定することで、管理されていない属性はサーバーに完全に認識され、すべてのコンテキストからアクセスできるようになります。既存のレルムを宣言型ユーザープロファイルに移行し始めていて、ユーザープロファイルの設定ですべてのユーザー属性がまだ定義されていない場合に役立ちます。 `管理者だけが書き込み可能` を設定することで、管理コンソールとAPIだけで管理できない属性は、既にユーザーが管理できる任意のカスタム属性を定義している場合に便利ですが、管理者が管理するべき他の属性を追加することを保証していません。 `管理者だけが参照可能` を設定することで、管理コンソールとAPIでのみ、管理対象の属性は読み取り専用でのみ利用できます。
+validateBindDn=LDAP管理者のDNを入力する必要があります
+useRefreshTokensHelp=これがonの場合、resh_tokenが作成され、トークンレスポンスに追加されます。これがoffの場合、resh_tokenは生成されません。
```

Comment:
@k-tamura - see below for a suggestion to use `refresh_token` like in the original English string. I added a suggestion in Weblate. 

https://hosted.weblate.org/translate/keycloak/-/ja/?q=has:suggestion#suggestions

cc: @y-tabata / @wadahiro